types/assert-equal-jsx/assert-equal-jsx-tests.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import {assertEqualJSX} from 'assert-equal-jsx';
```

Comment:
This package is using `module.exports = function assertEqualJSX`, which would be imported by `import assertEqualJSX = require("assert-equal-jsx")` in TypeScript and exported with `declare function assertEqualJsx(...); export = assertEqualJsx`. Be sure to test that this works at runtime.